interp/impargs.mli

```diff
@@ -119,6 +119,12 @@ val declare_manual_implicits : bool -> GlobRef.t -> ?enriching:bool ->
 val maybe_declare_manual_implicits : bool -> GlobRef.t -> ?enriching:bool ->
   manual_implicits -> unit
 
+(** [set_implicits local ref l]
+   Manual declaration of implicit arguments, for use by the Arguments command.
+   takes a list of possible sequences of implicit statuses, where
+   None = not implicit, Some false = non-maximal implicit, Some true = maximal implicit. *)
+val set_implicits : bool -> GlobRef.t -> bool option list list -> unit
```

Comment:
Why do we pass a bool option here instead of a Vernacexpr.vernac_implicit_status?

Reply:
That would probably be better, wouldn't it.
I think I was worried about layering issues, and also there was a point in drafting this PR when the type was `(Name.t * bool) option` and I didn't rethink the type when I removed the name.

Reply:
There was actually a layering issue, so I pulled the definition of `vernac_implicit_status` down into `Impargs` and left an alias in `Vernacexpr`.